source/plugin/text/representations/configurate.rst

```diff
@@ -8,23 +8,24 @@ Configuration Format
     ninja.leaping.configurate.loader.ConfigurationLoader
 
 SpongeAPI offers support to serialize text directly to a Configurate configuration file through the use of the
-``TypeToken`` class. :javadoc:`Text` objects are saved using the same node structure as the ``Text``\'s JSON
-representation.
+``TypeToken`` class. :javadoc:`Text` objects are saved using the same node structure as the ``Text``\'s 
+:doc:`json`, but in a more flexible format called :doc:`Human-Optimized Config Object Notation (HOCON) 
+</server/getting-started/configuration/hocon>`.
 
 .. tip::
 
-    For information on how to use Configurate to create configuration files for your plugin, please refer to
+    For information on how to use Configurate to create configuration files for your plugin, please refer to 
```

Comment:
Trailing whitespace

Reply:
Oh OK. I understand.

I've sorted out my branches and performed a rebase.  Hopefully, I have the system down now and won't be creating extra work. 😃